### PR TITLE
[FIX][web_widget_image_download] Make button look good everywhere (I hope).

### PR DESCRIPTION
**Pull request by yajo**
Fix https://github.com/OCA/web/pull/391#issuecomment-235906320.

This is some sort of backport from #392 

@Tecnativa

### CODE REVIEW

**Comment by yajo** [comment]
Linking this to https://github.com/odoo/odoo/pull/12918. It will fail functionally outside of OCB.

**Comment by pedrobaeza** [comment]
:+1:

File: web_widget_image_download/static/src/js/web_widget_image_download.js [hunk 1]
```diff
@@ -1,37 +1,39 @@
...
+
         has_custom_image: function () {
-            // check if the image of the widget is different from the default placeholder
-            return this.imgSrc && !this.imgSrc.includes('/placeholder.png');
-        }
+            return this.imgSrc != this.placeholder;
+        },
     });
-}
+});
```
Comment (lasley) [inline web_widget_image_download/static/src/js/web_widget_image_download.js]: This is causing the below in Runbot:

```
Uncaught SyntaxError: Unexpected token )
```

**Comment by lasley** [comment]
👍

**Pull request merged**